blivet/__init__.py

```diff
@@ -2134,6 +2134,9 @@ def getIgnoredNvdimmBlockdevs(self, nvdimm_ksdata):
         :rtype: set(str)
         """
 
+        if not self.nvdimm.nvdimm_plugin_available:
```

Comment:
I'd maybe add log.debug message saying we are returning empty set because the plugin is not available?

Reply:
Fixed.